docs/core/tutorials/testing-library-with-visual-studio.md

```diff
@@ -42,48 +68,15 @@ To create the unit test project, do the following:
-
-1. In the **Add New Project** dialog, select the **Visual Basic** node. Then select the **.NET Core** node followed by the **MSTest Test Project (.NET Core)** project template. In the **Name** text box, enter "StringLibraryTest" as the name of the project. Select **OK** to create the unit test project.
-
-   ![Add New Project dialog with unit test project displayed - Visual Basic](./media/testing-library-with-visual-studio/vb-create-new-test-project.png)
-
-   > [!NOTE]
-   > In addition to an MSTest Test project, you can also use Visual Studio to create an xUnit test project for .NET Core.
-
-1. Visual Studio creates the project and opens the *UnitTest1.vb* file in the code window.
-
-   ![Visual Studio code window for the unit test project class and method - Visual Basic](./media/testing-library-with-visual-studio/vb-unit-test-editor-window.png)
-
-   The source code created by the unit test template does the following:
-
-   - It imports the <xref:Microsoft.VisualStudio.TestTools.UnitTesting?displayProperty=nameWithType> namespace, which contains the types used for unit testing.
-
-   - It applies the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute> attribute to the `UnitTest1` class. Each test method in a test class tagged with the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute> attribute is executed automatically when the unit test is run.
-
-   - It applies the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute> attribute to define `TestMethod1` as a test method for automatic execution when the unit test is run.
-
 1. In **Solution Explorer**, right-click the **Dependencies** node of the **StringLibraryTest** project and select **Add Reference** from the context menu.
 
    ![Context menu of StringLibraryTest Dependencies](./media/testing-library-with-visual-studio/add-reference-context-menu.png)
 
 1. In the **Reference Manager** dialog, expand the **Projects** node and check the box next to **StringLibrary**. Adding a reference to the `StringLibrary` assembly allows the compiler to find **StringLibrary** methods. Select the **OK** button. This adds a reference to your class library project, `StringLibrary`.
 
-   ![Visual Studio add project reference dialog - Visual Basic](./media/testing-library-with-visual-studio/project-reference-manager.png)
-
----
+   ![Visual Studio add project reference dialog](./media/testing-library-with-visual-studio/project-reference-manager.png)
 
-## Adding and running unit test methods
+## Add and run unit test methods
 
 When Visual Studio runs a unit test, it executes each method marked with the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute> attribute in a unit test class, the class to which the  <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute> attribute is applied. A test method ends when the first failure is encountered or when all tests contained in the method have succeeded.
 
```

Comment:
You probably know better than I when it comes to this. I'll change.

Also, do you know where the code for the MSTest SDK comes from? I'm seeing some problems caused by the wrong xml tag being used. I imagine this SDK is auto-generated?!?

Reply:
I think the Assert.ThrowsException is preferred because you can control exactly where in the method you expect the exception to be thrown. With the attribute, the exception could be thrown in an unexpected place and the test would still pass.

For the MSTest SDK code, I'd ask @AbhitejJohn.